src/app/demo/bootstrap/bootstrap.component.html

```diff
@@ -7,6 +7,14 @@
   <app-mode-overview></app-mode-overview>
   <app-feedback-widget></app-feedback-widget>
 
+  <app-slide id="intro" class="no-padding">
+    <app-title-slide
+      [title]="title"
```

Comment:
Everything looks great, the only thing: can you have binding directly in the slides instead of having them in tho component? this way there's no need to jump between files. 

You'll have to get rid or square braces there, e.g.
title = "title"